arc/cli/loading_interface.py

```
@@ -0,0 +1,329 @@
+            SpinnerColumn(),
+            TextColumn("[progress.description]{task.description}"),
+            BarColumn(bar_width=LAYOUT["progress_bar_width"]),
+            TextColumn("[progress.percentage]{task.percentage:>3.0f}%"),
+            TimeElapsedColumn(),
+            console=self.console,
+            expand=False
+        )
+        
+        return progress
+    
+    def display_live_metrics(self, metrics: Dict[str, Any]) -> Panel:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)